src/client.rs

```diff
@@ -455,6 +455,24 @@ impl<T: DaprInterface> Client<T> {
             .collect();
         self.0.decrypt(requested_items).await
     }
+
+    /// Distributed lock request call
+    ///
+    /// # Arguments
+    ///
+    /// * `request` - Request to be made, TryLockRequest
+    pub async fn lock(&mut self, request: TryLockRequest) -> Result<TryLockResponse, Error> {
+        self.0.lock(request).await
+    }
+
+    /// Distributed lock request call
+    ///
+    /// # Arguments
+    ///
+    /// * `request` - Request to be made, TryLockRequest
+    pub async fn unlock(&mut self, request: UnlockRequest) -> Result<UnlockResponse, Error> {
+        self.0.unlock(request).await
+    }
```

Comment:
I think it would be beneficial to have examples here in the doc comments of building and using the api call e.g.
```
    /// # Example
    ///
    /// ```
    /// let result = client
    ///     .lock(dapr::client::TryLockRequest {
    ///         store_name: "lockstore".to_string(),
    ///         resource_id: "resource".to_string(),
    ///         lock_owner: "owner".to_string(),
    ///         expiry_in_seconds: 60,
    ///     })
    ///     .await
    ///     .unwrap();
    /// ```
```